templates/botnavs.js

```diff
@@ -8,20 +8,29 @@ export default function botnavs(section, locale) {
       [#-- Dev Sandbox / 2021_New_Clients_Banners_Map --][#contentmap id='dda18b65-5406-489b-a7f3-30539cb5a198'/]
     </td>
   </tr>` : ``}
+  <tr>
+    <td style="padding: 10px 0 0 0;">
```

Comment:
adding border so it can trigger dark mode for the top line?
```
  <tr>
    <td style="padding: 10px 0 0 0;" align="center" >
        <table width="662" height="3" border="0" bgcolor="black" cellspacing="0" cellpadding="0" align="center">
        <tr>
          <td style="font-size:1px;border-bottom:1px solid #000000;">&nbsp;</td>
        </tr>
      </table>
    </td>
  </tr>
```

Reply:
Hey, this didn't work for me in Outlook on iPhone, it does not trigger anything on the top line. Maybe I have a different version app (4.22)? 
But I totally agree we should fix it, Patrick if you have time can you explore more?

Reply:
yeah. it worked on mine and Katie's but it's not working on Patrick's neither. i tried using line text but it doesn't work neither. will need to explore further.